### ISSUE TITLE
spirit-project.com broken

### ISSUE THREAD

**Issue by sureshvv** [nitely/Spirit#314]
Demo site not working. Looks like domain has expired.

**Comment by nitely** [comment]
Yep, and it's not coming back. If you want to try it either install it or
try the one click Heroku deploy https://github.com/nitely/spirit-heroku

**Comment by sureshvv** [comment]
Thank you. Can we change the reference in the pypi page at https://pypi.org/project/django-spirit/ ?

**Comment by nitely** [comment]
done
